atqa-framework/utils/copyDirHandler.ts

```diff
@@ -0,0 +1,14 @@
+import fs from 'fs-extra';
```

Comment:
Why this file has a `Handler` suffix? What does it handle?

Reply:
handle dir? :D